koku/api/report/aws/provider_map.py

```diff
@@ -75,35 +76,42 @@ def __init__(self, provider, report_type):
                         'aggregates': {
                             'infrastructure_cost': Sum('unblended_cost'),
                             'derived_cost': Sum(Value(0, output_field=DecimalField())),
-                            'cost': Sum('unblended_cost')
+                            'markup_cost': Sum('markup_cost'),
+                            'cost': Sum(ExpressionWrapper(F('unblended_cost') + F('markup_cost'),
+                                                          output_field=DecimalField())),
                         },
                         'aggregate_key': 'unblended_cost',
                         'annotations': {
                             'infrastructure_cost': Sum('unblended_cost'),
                             'derived_cost': Value(0, output_field=DecimalField()),
-                            'cost': Sum('unblended_cost'),
+                            'markup_costs': Sum('markup_cost'),
+                            'cost': Sum(ExpressionWrapper(F('unblended_cost') + F('markup_cost'),
+                                                          output_field=DecimalField())),
                             'cost_units': Coalesce(Max('currency_code'), Value('USD'))
                         },
-                        'delta_key': {'cost': Sum('unblended_cost')},
+                        'delta_key': {'cost': Sum(ExpressionWrapper(F('unblended_cost') + F('markup_cost'),
+                                                                    output_field=DecimalField()))},
                         'filter': [{}],
                         'cost_units_key': 'currency_code',
                         'cost_units_fallback': 'USD',
-                        'sum_columns': ['cost', 'infrastructure_cost', 'derived_cost'],
+                        'sum_columns': ['cost', 'infrastructure_cost', 'derived_cost', 'markup_cost'],
```

Comment:
```suggestion
                        'sum_columns': ['cost', 'infrastructure_cost', 'derived_cost', 'markup_costs'],
```

I think you need the aliased column name here.